src/components/SearchSection/index.js

```diff
@@ -79,7 +79,7 @@ function SearchSection({
                 { name: "Covid-19", value: "Covid-19" },
                 { name: "Anti-vaxx", value: "Anti-vaxx" },
                 { name: "Foreign Influence", value: "Foreign Influence" },
-                { name: "None", value: "" },
+                { name: "None", value: "theme-none" },
```

Comment:
Are reason why empty string or `undefined` doesn't work?

Reply:
For some reason it causes a glitch in re-rendering when a user selects on any of the Selects i.e 
<img width="817" alt="Screenshot 2022-01-25 at 06 48 48" src="https://user-images.githubusercontent.com/12892109/150922732-b92232c5-9780-4db9-9018-ef33cf0a20c9.png">

Other Select Components are rendered, I assume it is because the value is the same.

Reply:
I'm playing with the currently deployed [instance](https://trolltracker.investigate.africa/explore?days=14) and I don't see this glitch @gertie-sheshe

Reply:
If you use `undefined` or an empty string it also does not show the value `None` on the Select @kilemensi

Reply:
Docs, docs, docs ... Yes, shouldn't use `undefined` unless you also use the `NativeSelect` component

But we're using empty string (`""`) and this should be fine. You can check the normal [example](https://v4.mui.com/components/selects/#simple-select) and native [example](https://v4.mui.com/components/selects/#native-select)... It won't show the None value if the Select already have a label. But between these two examples you should be able to figure out how to make it work with `""`.

Reply:
I have tried this with option and setting the native prop without label but still does not show None while using an empty string. I am beyond my thinking capacity with this and still settling for using a unique string @kilemensi to display the None value, unless you already know how to solve it then you can help me out and we can merge this PR

Reply:
Pushed a fix for this @gertie-sheshe